(docs)protocol-designer: PD 8.7 manual revisions (#20256)
OpenTrons/opentrons-api: docs/protocol-designer/docs/steps/transfer.md

* **Custom flow rate**: the speed the robot aspirates or dispenses liquid at. 
* **Well order**: the order the robot addresses source or destination wells in. 
* **Tip position**: where the robot aspirates or dispenses in your labware. 
* Other pipetting settings like submerge and retract speed, mix, delay, blowout, and air gap.  

> If you chose to apply a liquid class to your transfer step, each tab already contains values, like flow rate, optimized to transfer your liquid. In this form, you're able to enter custom values in the valid range for settings like tip position. Default values and ranges change depending on the combination of pipette and tips selected to perform the transfer. Click at the bottom of either tab to remove your custom settings at any time. 

Reviewer: I would like to still include the text **Reset to default** here. Otherwise isn't unclear exactly what control you need to click on.

Author: The text has been changed to **reset aspirate settings** or **reset dispense settings** but I agree. I'll try it a few more ways.

* **Tip position**: where the robot aspirates or dispenses in your labware. 
* Other pipetting settings like submerge and retract speed, mix, delay, blowout, and air gap.  

> If you chose to apply a liquid class to your transfer step, each tab already contains values, like flow rate, optimized to transfer your liquid. In this form, you're able to enter custom values in the valid range for settings like tip position. Default values and ranges change depending on the combination of pipette and tips selected to perform the transfer. Click at the bottom of either tab to **Reset aspirate settings** or **Reset dispense settings**. 

Reviewer: Aha, I see what changed in the interface here. Good edit.

- the **start point** in the labware to begin the submerge or retract from. 

Additional advanced pipetting settings are available in the Aspirate and Dispense tabs. These are listed in the order in which the robot performs them. Protocol Designer supports the following advanced settings: 

| **Advanced Setting** | **Pipette Movement** | **Description** |
| :------------------- | :------------------- | :-------------- |
| Pre-wet tip | <ul><li>Aspirate</li></ul> | <ul><li>Aspirate and dispense once in the source well before aspirating the transfer volume</li><li>Takes place at your chosen aspirate position</li></ul> |
| Mix | <ul><li>Aspirate</li><li>Dispense</li></ul> | <ul><li>Mix the contents of the well either before aspirating or after dispensing</li><li>Customize volume and number of repetitions</li><li>Can take place at your chosen aspirate or submerge position</li></ul> |
| Delay | <ul><li>Aspirate</li><li>Dispense</li></ul> | <ul><li>Hold the pipette tip at the submerge, aspirate or dispense, or retract position for a defined amount of time after aspirating or dispensing</li><li>Customize duration and position from bottom of well</li></ul> |

> | Condition | <ul><li>Aspirate</li></ul> | <ul><li>Aspirate a small conditioning volume after aspirating the total volume to be transferred</li><li>Pipetted back into the source for a more accurate first dispense</li><li>Only available for distribute, or multi-dispenses</li></ul> |

Author: @ncdiehl11 and/or @jerader : 

First time adding more technical descriptions of conditioning volume, disposal volume, and push out to the PD manual. please check my accuracy on conditioning/disposal volume especially - we tend to keep these kind of quiet on the API docs side, but here users already get a small chunk of info text in PD so I added to it.

Reviewer 2: For pushout, maybe something more aligned with our API documentation, which specifies: 
```
Continue past the plunger bottom to help ensure all liquid leaves the tip
```

Reviewer 2: For disposal volume, that's actually volume aspirated _before_ the dispense sequence, to ensure each dispense in the sequence receives its specified volume of liquid (with some leftover)

Author: Good catch on disposal volume! 

I updated push out : `Dispense past the pipette's plunger bottom to ensure all liquid leaves the tip.`

since the push out distance to move past the plunger bottom is calculated with a volume, I want to continue tying those principles together for users.

* **Per destination**: selects a new tip for each destination well, including the first. Not compatible with a consolidate or distribute path. 

Consolidate and distribute paths use a cycle of aspiration and dispensing to combine liquid in source or destination wells. When you choose "always" for a consolidate or dispense path, the robot selects a new tip before the first aspirate of the cycle: 

* Consolidate cycle: aspirate, aspirate, dispense.
* Distribute cycle: aspirate, dispense, dispense. 

If you choose an incompatible well ratio or an empty source well, Protocol Designer displays a warning in the transfer form. 

<figure class="screenshot" markdown>
  ![Transfer warning](../images/transfer_warning.png)
  <figcaption>Incompatible well ratio in a transfer step.</figcaption>
</figure>

Next, choose a tip drop location to dispose of used tips before clicking **Continue**. Options include fixtures like the trash bin and waste chute, if used in your protocol, or to return tips to the tip rack. If you select **Tip rack** as the tip drop location, the pipette will return tips to their original position in the tip rack. 

> You can also customize tip tracking settings in your Protocol Designer protocols. When you select automatic tip tracking, Protocol Designer keeps track of which tips have been picked up and used, starting with the tip in position A1 of the tip rack you chose for the transfer. 

Author: @ncdiehl11 and/or @jerader : is this accurate of automatic tip tracking? am I being too specific? I could always go simpler, like:: 

> Protocol Designer tracks which tips have been picked up and used, and selects the next available tip in the tip rack.

Author: General review of the new *Tip management* and *Partial tip use* sections in `transfer.md` is helpful too!!

Reviewer 2: I think that this is great! Maybe mentioning the starting tip position can be omitted, since depending on your pipette/nozzle configuration, this can refer to the A1, A1-H1, or A1-H12

<figure class="screenshot" markdown>
  ![Manual tip selection](../images/tip_selection.png)
  <figcaption>Click to select tips for your transfer step.</figcaption>
</figure> 

Protocol Designer includes warnings if your tip selections might be incompatible with other parts of your protocol, especially in partial tip use. See below for more.

## Partial tip use

You can use partial tip pickup to use less tips than a Flex or OT-2 multi-channel pipette can pick up at once: 

* Pick up a single column or individual tip with the Flex 96-channel pipette. 
* Pick up individual tips with Flex and OT-2 8-channel pipettes. 

When you add a transfer or mix step, start by selecting **column** or **single** from the available nozzle options in the first form.

!!! Note
    You'll need to place tip racks directly on the starting deck for partial tip pickup with the Flex 96-channel pipette. 

    When you select the Flex 96-channel pipette, Protocol Designer automatically places your first tip rack on the deck with a 96-channel tip rack adapter, required for picking up a full rack of tips. Add a second tip rack directly on the deck to enable partial tip use.

> In the fourth form, you can choose between automatic and manual tip tracking for your multi-channel pipette. Protocol Designer includes warnings, both in the tip selection form and as a step error, if your partial tip selections could cause pipette collisions. We always recommend resolving all errors before exporting your protocol and running it on a robot. 

Author: @ncdiehl11 and/or @jerader : Added this based on some warnings I noticed when messing around in the sandbox. If there's any further guidance for users, I could add some of it here.

Reviewer 2: This makes sense to me. I don't think I have much to add. The only thing I can think is that some users may be used to platforms where the nozzles can be controlled independently, but on our platform, the pipette moves all nozzles in unison, so we have to be mindful about adjacent labwares'/modules' heights during partial tip pickup.

This might be more info than is actually useful though. What you have is concise and clear

Author: Made some updates in my next commit.

The default tip position value of 0 represents the middle of the well for both X and Y positions. In the example above, a positive X value moves the tip to the right within the well, a negative Y value moves the tip to the left, and a positive Z value moves the tip up towards the top of the well. as you enter custom values, the graphic changes to demonstrate the new tip position. Toggle between top and side views of the well by clicking **Swap view**.

For an aspirate or dispense, Protocol Designer lets you customize submerge and retract settings:

- the **speed** the pipette will submerge into or retract from the liquid. 
- whether the pipette should **delay** before submerging or retracting.

> - the **start point** in the labware to begin the submerge or retract from. 

Reviewer: caps for list items. and i think italics here, since these are not controls you click on.
```suggestion
- The *speed* the pipette will submerge into or retract from the liquid. 
- Whether the pipette should *delay* before submerging or retracting.
- The *start point* in the labware to begin the submerge or retract from. 
```

<figure class="screenshot" markdown>
  ![Transfer warning](../images/transfer_warning.png)
  <figcaption>Incompatible well ratio in a transfer step.</figcaption>
</figure>

Next, choose a tip drop location to dispose of used tips before clicking **Continue**. Options include fixtures like the trash bin and waste chute, if used in your protocol, or to return tips to the tip rack. If you select **Tip rack** as the tip drop location, the pipette will return tips to their original position in the tip rack. 

You can also customize tip tracking settings in your Protocol Designer protocols. When you select automatic tip tracking, Protocol Designer keeps track of which tips have been picked up and used. You can click **manual tip tracking** to manually select the tips the pipette will use in your transfer step. Tips in your tip rack are labeled as:

* **New** tips haven't been used yet.
* **Used** tips have been used and returned to the tip rack.
* **Selected** tips will be used in your transfer. Tips are numbered to show how many will be picked up at once.
* Tips labeled **no tip** have been discarded and are missing from the tip rack.

> * **Inaccessible** tips can't be picked up by your chosen pipette.

Reviewer: Here I think we can keep bold formatting (internal heading of list items) but add colons and make sure the bolded item always goes first:
```suggestion
* **New**: Tips that haven't been used yet.
* **Used**: Tips that have been picked up and returned to the tip rack.
* **Selected**: Tips to be used in your transfer. Tips are numbered to show how many will be picked up at once.
* **No tip**: An empty location in the tip rack. This means the tip from that location has been picked up and discarded.
* **Inaccessible**: Tips that can't be picked up by your chosen pipette or partial nozzle configuration.
```

<figure class="screenshot" markdown>
  ![Manual tip selection](../images/tip_selection.png)
  <figcaption>Click to select tips for your transfer step.</figcaption>
</figure> 

Protocol Designer includes warnings if your tip selections might be incompatible with other parts of your protocol, especially in partial tip use. See below for more.

> ## Partial tip use

Reviewer: We call this `Partial tip pickup` everywhere else, so we might as well here too.

* Pick up a single column or individual tip with the Flex 96-channel pipette. 
* Pick up individual tips with Flex and OT-2 8-channel pipettes. 

When you add a transfer or mix step, start by selecting **column** or **single** from the available nozzle options in the first form.

!!! Note
    You'll need to place tip racks directly on the starting deck for partial tip pickup with the Flex 96-channel pipette. 

    When you select the Flex 96-channel pipette, Protocol Designer automatically places your first tip rack on the deck with a 96-channel tip rack adapter, required for picking up a full rack of tips. Add a second tip rack directly on the deck to enable partial tip use.

In the fourth form, you can choose between automatic and manual tip tracking for your multi-channel pipette. Protocol Designer includes warnings, both in the tip selection form and as a step error, if your partial tip selections could cause pipette collisions. 

<figure class="screenshot" markdown>
  ![Pipette collision risk](../images/partial-tip-collision.png)
  <figcaption>Labware in adjacent slots can cause pipette collisions.</figcaption>
</figure> 

> Partial tip pickups require the pipette to hover above adjacent slots. Here, Protocol Designer won't allow you to select tips for manual tip tracking, because the tip racks in adjacent deck slots could cause a pipette collision. Despite the tip rack being full of pipette tips, the "not enough tips" warning tells you that there aren't enough pipette tips that the pipette can pick up safely. This also results in a pipette collision error. 

Reviewer: ```suggestion
Partial tip pickup requires the pipette to hover above adjacent slots. Here, Protocol Designer won't allow you to select tips for manual tip tracking, because the tip racks in adjacent deck slots could cause a pipette collision. Despite the tip rack being full of pipette tips, the "not enough tips" warning tells you that there aren't enough pipette tips that the pipette can pick up safely. This also results in a pipette collision error. 
```
